Source/DafnyDriver/DafnyDriver.csproj

```diff
@@ -5,10 +5,11 @@
       <AssemblyName>Dafny</AssemblyName>
       <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
       <DefineConstants>TRACE</DefineConstants>
+      <TargetFramework>net5.0</TargetFramework>
+      <OutputPath>..\..\Binaries\</OutputPath>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netcoreapp3.1|AnyCPU'">
-    <OutputPath>..\..\Binaries\</OutputPath>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net5.0|AnyCPU'">
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
   </PropertyGroup>
```

Comment:
Does this PropertyGroup need an OutputPath?

Reply:
This PropertyGroup applies only to Debug|netcoreapp3.1|AnyCPU. But we want OutputPath to be set, regardless of the configuration|framework|platform. Therefore, I moved the OutputPath item from here to the unconditional PropertyGroup above.